packages/components/src/Modal/ContextModal/_docs/ContextModal.stories.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from "react"
+import { Decorator, Meta, StoryObj } from "@storybook/react"
+import isChromatic from "chromatic"
+import { AddImage } from "@kaizen/draft-illustration"
+import { ModalDescription } from "~components/Modal/GenericModal/subcomponents/ModalDescription"
+import { Text } from "~components/Text"
+import { ContextModal, ContextModalProps } from "../index"
+
+const IS_CHROMATIC = isChromatic()
+
+const ContextModalWithState = (args: ContextModalProps): JSX.Element => {
```

Comment:
As with the other PR (optional): Converting to a Story might be better for intellisense.